tests/test_chargebee_all_fields.py

```diff
@@ -0,0 +1,316 @@
+        'credit_notes': {
+            'voided_at',
+            'vat_number',
+            'discounts'
+        },
+        'items': {
+            'archivable',
+            'gift_claim_redirect_url',
+            'applicable_items',
+            'redirect_url',
+            'usage_calculation',
+            'included_in_mrr'
+        },
+        'coupons': {
+            'invoice_notes',
+            'meta_data',
+            'archived_at'
+        },
+        'customers': {
+            'backup_payment_source_id',
+            'meta_data',
+            'cf_company_id',
+            'custom_fields',
+            'created_from_ip',
+            'consolidated_invoicing',
+            'billing_day_of_week',
+            'vat_number'
+        },
+        'subscriptions': {
+            'cancel_reason',
+            'start_date',
+            'meta_data',
+            'remaining_billing_cycles',
+            'payment_source_id',
+            'custom_fields',
+            'item_tiers',
+            'invoice_notes',
+            'created_from_ip',
+            'cancel_reason_code',
+            'coupon',
+            'coupons'
+        },
+        'transactions': {
+            'error_text',
+            'reference_number',
+            'error_code',
+            'refunded_txn_id'
+        },
+        'promotional_credits': {
+            'reference'
+        },
+        'events': {
+            'user'
+        }
+    }
+    # fields for V1
+    fields_to_remove_V1 = {
+        'coupons': {
+            'included_in_mrr'
+        },
+        'addons': {
+            'avalara_service_type',
+            'accouting_category1',
+            'accouting_category3',
+            'taxjar_product_code',
+            'accouting_category4',
+            'avalara_transaction_type',
+            'tiers',
+            'accouting_category2',
+            'tax_code',
+            'price_in_decimal',
+            'included_in_mrr',
+            'tax_profile_id',
+            'avalara_sale_type'
+        },
+        'quotes': {
+            'contract_term_start',
+            'line_item_tiers',
+            'vat_number_prefix',
+            'invoice_id',
+            'contract_term_termination_fee',
+            'contract_term_end'
+        },
+        'plans': {
+            'avalara_service_type',
+            'account_code',
+            'event_based_addons',
+            'free_quantity_in_decimal',
+            'taxjar_product_code',
+            'applicable_addons',
+            'accounting_category4',
+            'avalara_transaction_type',
+            'claim_url',
+            'tiers',
+            'tax_profile_id',
+            'tax_code',
+            'accounting_category3',
+            'price_in_decimal',
+            'archived_at',
+            'attached_addons',
+            'avalara_sale_type',
+            'trial_end_action'
+        },
+        'subscriptions': {
+            'offline_payment_method',
+            'gift_id'
+        }
+    }
```

Comment:
Please add comments with the reason for each field like why we have to skip it.  Add a comment at the map level if the reason is the same for all fields on the map.

Reply:
Added comment for removed fields during assertion.